include/openssl/base.h

```diff
@@ -203,6 +203,11 @@ extern "C" {
 
 #define AWSLC_API_VERSION 16
 
+// In order to keep track of any changes to AWLC we introduce a string that tracks with the public Github
+// release version at https://github.com/awslabs/aws-lc
+
+#define AWSLC_VERSION_NUMBER_STRING "0.0.2"
```

Comment:
Do we also need a version string that can be processed at compile-time? You can't really parse this string. Similar to `OPENSSL_VERSION`

Reply:
I have this version string so that we only need to change this in one place to get the code change done (we also need to change the tests but I think this simplifies things)

Reply:
I don't mind having a version string and a version number.
How do you parse this string at compile-time?
With a version number you can branch at compile-time e.g. https://github.com/aws/s2n-tls/blob/main/crypto/s2n_openssl.h#L36
I dunno if you can do something similar with the string version. Or we need another mechanism.